### PR TITLE
Dockerfile update

### PR DESCRIPTION
**Pull request by kylma**
Update of the go version, added go tarball signature verification

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,15 +6,21 @@ RUN apt-get install -y \
     git gnupg make parted qemu-user-static wget xz-utils zip \
     debootstrap sudo dirmngr bison flex libssl-dev kmod udev cpio
 
+# import U-Boot signing keys
+RUN gpg --batch --keyserver hkp://ha.pool.sks-keyservers.net --recv-keys 38DBBDC86092693E && \
+    gpg --batch --keyserver hkp://ha.pool.sks-keyservers.net --recv-keys 147C39FF9634B72C && \
+# import golang signing keys
+    gpg --batch --keyserver hkp://ha.pool.sks-keyservers.net --recv-keys 'EB4C 1BFD 4F04 2F6D DDCC EC91 7721 F63B D38B 4796'
```
Comment (eduncan911) [inline Dockerfile:13]: Why not use the short form format like the other two keys?

**Review by eduncan911 — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (kylma) [reply Dockerfile:13]: I forgot a commit... Should I do another PR since this one was merged?

Comment (eduncan911) [reply Dockerfile:13]: I'm just a friendly commuter of PRs, such as yourself.

If they merged it as-is, so be it!  Hehe.

IOW, nah. It was just a suggestion before they merged it.

Comment (andrejro) [reply Dockerfile:13]: @kylma, sure go ahead.

**Pull request merged**